privacy-and-messaging.md

```diff
@@ -0,0 +1,138 @@
+
+
+## Real examples of security breaches or issues that have occurred with these applications
+
+Jeff Bezos' phone was famously hacked in January 2020 through a WhatsApp video message. In December of the same year, Texas' attorney general alleged that Facebook and Google struck a back-room deal to reveal WhatsApp message content. A spyware vendor targeted a WhatsApp vulnerability with its software to hack 1,400 devices, resulting in a lawsuit from Facebook. WhatsApp's unencrypted cloud-based backup feature has long been considered a security risk by privacy experts.  
+
+When WhatsApp says it can't view the content of the encrypted messages you send to another WhatsApp user, what it doesn't say is that there's a list of other data that it collects that could be linked to your identity: Your unique device ID, usage and advertising data, purchase history and financial information, physical location, phone number, your contact information and that of your list of contacts, what products you've interacted with, how often you use the app, and how it performs when you do. The list goes on. This is way more than Signal or Telegram. 
+
+WhatsApp Privacy Leak Possibility: In 2019 Check Point Research (CPR) revealed that the popular messaging app, WhatsApp, could also be tweaked to reveal user data and conversations. :
+https://research.checkpoint.com/2019/breakingapp-whatsapp-crash-data-loss-bug/
+
+No, your privacy is not safe on messaging apps. Here’s why: 
+https://medium.com/@secretum/no-your-privacy-is-not-safe-on-messaging-apps-heres-why-43b3eb02875
+
+Recently Signal has come under scrutiny 'Elon Musk Fact-Checked On X After Secure Messaging Warning' :
+https://www.forbes.com/sites/zakdoffman/2024/05/08/elon-musk-fact-checked-on-x-after-surprise-messaging-warning/?sh=688389ec6ccd
+
+
+## References:
+
+A good comparison website on the privacy of messaging apps:
+https://www.securemessagingapps.com/
+
+Leaks and why certain apps shouldn't be trusted:
+https://www.securemessagingapps.com/ratings/
+
+Comparison of Cross Platform Instant Messaging Clients:
+https://en.wikipedia.org/wiki/Comparison_of_cross-platform_instant_messaging_clients
+
+Study Rates Signal Most Secure Messaging App, But Should Users Be Worried?:
+https://techround.co.uk/tech/signal-secure-messaging-app-users-worry/
+
+Line mobile messaging app hit by systems breach, thousands of accounts leaked:
+https://www.techradar.com/pro/security/line-mobile-messaging-app-hit-by-systems-breach-thousands-of-accounts-leaked
+
+
+## Conclusion:
+
+Users seeking to protect their personal information should be aware of how these messaging platforms work and what data is private. Signal and Bison Relay stand out from the crowd when it comes to privacy. Users must remain vigilant and informed about the privacy policies and security features of the messaging apps they choose to use.
+
```

Comment:
Well structured and reads well. This piece acts as a very good overview of the types of messaging apps and the data they collect. It also highlighst privacy concerns with these apps, some of which I wasn't familiar with before reading this piece.

Thoughts for future improvements. It's fair to say that your writing and content has improved massively this past year. To take it to the next level, it would be good to pursue areas of critical thinking and enquiry, including data analysis and questioning.

For instance, what questions would you like to find the answers to and what are the counter arguments. For example, Line 55, could have been extended and elaborated on.

"Concerns however have been raised over metadata collection and how this plays out in Meta's broader ecosystem."

The questions I'd ask for this particular statement include:

* what concerns have been raised and why?
* What metadata is being collected?
* Why is it being collected?
* What is the company doing with this data?

I hope this feed back helps, once you are ready to publish, feel free to upload to CT.

Reply:
Thank you very much. That's good to hear. I feel I have learnt a lot on this journey and hope to get better. I'm going to add another article that covers the data https://www.wired.com/story/whatsapp-instagram-facebook-data/#:~:text=Data%20WhatsApp%20can%20share%20with,%2Dcalled%20%E2%80%9CFacebook%20companies%E2%80%9D.

I'll add to ghost now